pyvera/__init__.py

```diff
@@ -490,6 +495,10 @@ def vera_device_id(self):
         """The ID Vera uses to refer to the device."""
         return self.device_id
 
+    @property
+    def should_poll(self):
```

Comment:
Nice approach - assume you'll use the in ths HA wrapper code?

I think there are a few other components that depend on polling (I think locks, but we should check so these are set correctly in the Vera devices). Basically anything that uses `get_complex_value` will need polling.